tests/backup/backup_test.go

```diff
@@ -6498,36 +6508,39 @@ var _ = Describe("{ScheduleBackupCreationSingleNS}", func() {
 				dash.VerifyFatal(err, nil, fmt.Sprintf("Verification of adding backup location - %s", backupLocationName))
 			}
 		})
-
 		Step("Creating Schedule Policies", func() {
 			log.InfoD("Creating Schedule Policies")
 			periodicSchedulePolicyInfo := Inst().Backup.CreateIntervalSchedulePolicy(5, 15, 5)
-			periodicPolicyStatus := Inst().Backup.BackupSchedulePolicy(periodicPolicyName, uuid.New(), orgID, periodicSchedulePolicyInfo)
-			dash.VerifyFatal(periodicPolicyStatus, nil, fmt.Sprintf("Verification of creating periodic schedule policy - %s", periodicPolicyName))
+			periodicSchedulePolicyUID := uuid.New()
+			err := Inst().Backup.BackupSchedulePolicy(periodicPolicyName, periodicSchedulePolicyUID, orgID, periodicSchedulePolicyInfo)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verification of creating periodic schedule policy - %s", periodicPolicyName))
 		})
-
 		Step("Adding Clusters for backup", func() {
-			log.InfoD("Adding application clusters")
+			log.InfoD("Adding Clusters for backup")
```

Comment:
nit: change `Clusters` to `clusters`

Reply:
I will change this when I make a PR for this test case separately.